src/Styleguide/Components/Pagination.tsx

```diff
@@ -8,7 +8,10 @@ import { Responsive } from "Utils/Responsive"
 import { Pagination_pageCursors } from "__generated__/Pagination_pageCursors.graphql"
 import { createFragmentContainer, graphql } from "react-relay"
 
+const PAGE_NUMBER_CAP = 100
+
 interface Props {
+  displayLast?: boolean
```

Comment:
I'm not sure about the name since we still want to show the link to the last page when e.g. it only have 10 pages. Open to suggestions and opinions.

Reply:
Possibly worth keeping in mind that this is a generic pagination control, whereas the 100 cap is more b/c of the specific implementation of V1 endpoints in gravity.

I was thinking this might be better handled in Metaphysics, where this component wouldn't even have to change. This component can then stay simple in terms of simply rendering Metaphysics pagination metadata, and wiring up prev/next/page click handlers.

Not a blocker at all, just musing. We can cap the Metaphysics `last` result to 100. Then, this prop can stay `displayLast` and it simply hides (or renders) the last page data. That could be passed in from a parent component, like `displayLast={totalPages === 100}` or something. We could even augment the metadata Metaphysics returns to say `capped: true` as signal to UI's that the list has been truncated, etc.

Reply:
We decided to go with completely hiding the last page button specifically for collect as we don't want to give the user the false sense that there are only 100 pages of results which isn't correct.

That being said, it makes sense to cap the page number to 100 in Metaphysics. In terms of the naming and type, `displayLastPage: boolean` makes sense since we want an on and off switch.

The `maxPages` option would also be a useful one but in this case this isn't the wanted behavior